Makefile

```diff
@@ -38,6 +38,7 @@ testdockerfile_%:
 	docker run --rm -i hadolint/hadolint < $(call dockerfileFromTag,$*)
 
 testentrypoint_%: 
+	shellcheck --version
```

Comment:
is this intentional?

Reply:
yes this was intentional as I found no good way to see what version of shellcheck was being used so that I could test locally first.   If you are aware of a better way that could be documented in the BUILD.md, otherwise I think this is useful in the event that travis-ci updates their embedded version.  (ps entrypoint will start to throw a bunch more errors as soon as they increment the shellcheck version to the next minor version)

Reply:
Oh! I see what you mean, then FWIW I guess the right place should be the `.travisci` which is the one with the concern of installation. When working locally this should not be a concern.
But please disregard, I'll take care of it later.